tracing/opentracing/client_interceptors.go

```diff
@@ -26,7 +27,7 @@ func UnaryClientInterceptor(opts ...Option) grpc.UnaryClientInterceptor {
 		}
 		newCtx, clientSpan := newClientSpanFromContext(parentCtx, o.tracer, method)
 		err := invoker(newCtx, method, req, reply, cc, opts...)
-		finishClientSpan(clientSpan, err)
+		finishClientSpan(newCtx, clientSpan, err)
```

Comment:
You'll need to attach the trace and span ids before you call `invoker` otherwise the context that is passed into the invoker will not have any of the opentracing information.

Reply:
The context will have the `traceid` and `spanid` inside the tag struct. `newClientSpanFromContext` makes sure that the values are there.